src/Router.php

```diff
@@ -28,6 +28,9 @@ public static function fromServerlessConfig(array $serverlessConfig): self
         foreach ($serverlessConfig['functions'] as $function) {
             $pattern = $function['events'][0]['httpApi'] ?? null;
             if (! $pattern) continue;
+            if (is_array($pattern) && isset($pattern['method']) && isset($pattern['path'])) {
```

Comment:
@chatii , Can we do a for-loop to support multiple events?

```
    events:
      - httpApi:
          method: PUT
          path: '/foo'
      - httpApi:
          method: GET
          path: '/foo'
```

Reply:
@hungvietdo 
It did not appear to be supported originally.
Modified to interpret multiple `httpApi` (both 'mapping' and 'string').